modules/i18n/src/translations/en-US/portals/ai.ts

```diff
@@ -0,0 +1,77 @@
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { aiNS } from "../../../models";
+
+export const ai: aiNS = {
+
+    aiLoginFlow:{
+        banner:{
+            collapsed:{
+                button: "Try Login AI",
+                heading: "Configure login flow with Login AI",
+                subheading: "Simple and effortless login flow configuration experience."
+            },
+            full:{
+                button: "Try Login AI",
+                heading: "Generate your login flow with a single click using Login AI",
+                subheading1: "Create your login sequence effortlessly with our intuitive AI.",
+                subheading2: "Enjoy simple and hassle-free configuration experience."
+            },
+            input:{
+                button: "Generate Login Flow",
+                heading: "Configure login flow with Login AI",
+                placeholder: "Example prompt:  Users should be prompted with a username-password login. " +
+                "Once they successfully login, I want the system to check if they belong to either the 'manager' " +
+                "or 'employee' group. If they're not part of any of these groups, I need them to be redirected to " +
+                "an error page with an error message",
+                subheading:
+                "Simple and effortless login flow configuration experience."
+            }
+        },
+        screens:{
+            loading:{
+                facts:{
+                    0:"WSO2 Adaptive Authentication supports adaptive multi-factor authentication," +
+                    " allowing organizations to dynamically adjust the level of authentication required based" +
+                    " on risk factors such as user location, device, and behavior.",
+                    1:"WSO2 Adaptive Authentication seamlessly integrates with various identity providers,"+
+                    " allowing organizations to leverage existing authentication mechanisms while"+
+                    " enhancing security with adaptive capabilities.",
+                    2:"WSO2 Adaptive Authentication features a flexible policy engine that enables organizations"+
+                    " to define custom authentication policies based on their specific security requirements and"+
+                    " regulatory compliance needs."
+                },
+                heading: "Generating your login flow",
+                states:{
+                    0: "Initalizing.....",
```

Comment:
Let's not use "...." to end the states and simply keep the text only.

WDYT @savindi7

Reply:
Changed the content as per the content revision for AI features. Please find the document [here](https://docs.google.com/document/d/1fBRt9RaBrUVYh-Dh0xKIGCzUlTej6wTwLoY5BlI78-0/edit?usp=sharing).

Reply:
In the document it has only three dots right ?, let's keep it that way if possible

Reply:
Using three dots is the standard.

Reply:
> In the document it has only three dots right ?, let's keep it that way if possible

Yes. 
Made the necessary changes.